pubsub/server_test.go

```diff
@@ -49,8 +49,6 @@ func (x *counter) dummyProcessTXCallback(b []byte, _ *Connection) {
 // a msg to be sent to all connections. Checks the message was delivered properly
 // and the connection is properly handled when closed.
 func TestServerPublish(t *testing.T) {
-	t.Skip("FLAKY")
```

Comment:
Removing `FLAKY` here in favor of seeing and fixing the test failures, since this is causing problems